src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -122,6 +189,116 @@ def _load_from_data(self, data):
                 if self._should_include_example(d):
                     self.examples.append(HelpExample(d))
 
+    # get data object from parsed yaml
+    def _load_from_parsed_yaml(self, data):
+        content = data.get("content")
+        info_type = None
+        info = None
+        new_data = {}
+
+        for elem in content:
+            for key, value in elem.items():
+                # find the command / group's help text
+                if value.get("name") and value.get("name") == self.command:
+                    info_type = key
+                    info = value
+                    break
+            if info:
+                break
+
+        # if a new command not found return old data object
+        if not info:
+            return data
+
+        new_data["type"] = info_type
+
+        if "summary" in info:
+            new_data["short-summary"] = info["summary"]
+
+        if "description" in info:
+            new_data["long-summary"] = info["description"]
+
+        if "examples" in info:
+            new_data["detailed_examples"] = info["examples"]
+            reg_examples = []
+            for item in new_data["detailed_examples"]:
+                ex = {}
+                ex["name"] = item.get("summary", "")
+                ex["text"] = item.get("description", "")
+                ex["text"] = "{}\n{}".format(ex["text"], item.get("command", "")) if ex["text"] else item.get("command", "")
+                ex["min_profile"] = item.get('min_profile')
+                ex["max_profile"] = item.get('max_profile')
+                reg_examples.append(ex)
+            new_data["examples"] = reg_examples
+
+        if "links" in info:
+            text = self._get_links_as_text(info["links"])
+            new_data["links"] = info["links"]
+            new_data["long-summary"] = "{}\n{}".format(new_data["long-summary"], text) \
+                if new_data.get("long-summary") else text
+
+        if "arguments" in info:
+            new_data["parameters"] = []
+            for arg_info in info["arguments"]:
+                new_data["parameters"].append(self._get_parameter_info(arg_info))
+
+        return new_data
+
+    @staticmethod
+    def _get_links_as_text(links):
+        text = ""
+        for link in links:
+            if "name" in link and "url" in link:
+                text += "- {}: {}.\n".format(link["name"], link["url"])
```

Comment:
Name should likely be renamed `title` since that's the attribute in the HTML tag that corresponds to the tooltip (which is presumably what docs team would use it for). Also, it makes clear that this would be a longer thing than a simple name.

Reply:
My opinion is that rather than pull this out into a semi-bulleted list, we do:
``` For more information, visit: <url> ``` and the title wouldn't be used.